scrapy/utils/python.py

```diff
@@ -174,17 +174,25 @@ def new_method(self, *args, **kwargs):
         return cache[self]
     return new_method
 
+
 _BINARYCHARS = {six.b(chr(i)) for i in range(32)} - {b"\0", b"\t", b"\n", b"\r"}
 _BINARYCHARS |= {ord(ch) for ch in _BINARYCHARS}
 
-
+@deprecated("scrapy.utils.python.binary_is_text")
 def isbinarytext(text):
-    """Return True if the given text is considered binary, or False
-    otherwise, by looking for binary bytes at their chars
+    """ This function has been removed.
+    Please use scrapy.utils.python.binary_is_text, which was created to be more
+    clear about the functions behavior: it is behaving inverted to this one. """
+    return not binary_is_text(text)
```

Comment:
@nyov I like the docstring. Just one nitpick: perhaps it's better to change "This function has been removed" into "This function is deprecated", since it is not removed yet.